middie.js

```diff
@@ -26,10 +26,20 @@ function middie (complete) {
       })
     }
 
-    middlewares.push({
-      regexp,
-      fn: f
-    })
+    if (Object.prototype.toString.call(f) === '[object Array]') {
```

Comment:
`Array.isArray`

Reply:
Sure. Was not sure how es next-ish you are :-)